src/pyuvdata/uvbeam/uvbeam.py

```diff
@@ -3678,8 +3879,14 @@ def read_cst_beam(
                 history = settings_dict["history"]
             if reference_impedance is None and "ref_imp" in settings_dict:
                 reference_impedance = float(settings_dict["ref_imp"])
-            if x_orientation is None and "x_orientation" in settings_dict:
-                x_orientation = settings_dict["x_orientation"]
+            if x_orientation is None:
+                x_orientation = settings_dict.get("x_orientation", None)
+            if feed_array is None:
+                feed_array = settings_dict.get("feed_array", None)
+            if feed_angle is None:
+                feed_angle = settings_dict.get("feed_angle", None)
+            if mount_type is None:
+                mount_type = settings_dict.get("mount_type", None)
```

Comment:
I think these should also be added to the `overriding_keywords` so there's a warning if they are set in the yaml and when the method is called.

Reply:
Alright, finally tackled this orphaned comment in 602c12b9fe74c2c43eda7a5595486ae763c6fa69